AMM/contracts/exchange/src/main.sw

```diff
@@ -0,0 +1,423 @@
+contract;
+
+dep errors;
+dep events;
+dep utils;
+
+use errors::{InitError, InputError, TransactionError};
+use events::{
+    AddLiquidityEvent,
+    DefineAssetPairEvent,
+    DepositEvent,
+    RemoveLiquidityEvent,
+    SwapEvent,
+    WithdrawEvent,
+};
+use libraries::{
+    data_structures::{
+        PoolInfo,
+        PreviewAddLiquidityInfo,
+        PreviewSwapInfo,
+        RemoveLiquidityInfo,
+    },
+    Exchange,
+};
+use std::{
+    block::height,
+    chain::auth::{
+        AuthError,
```

Comment:
Is this needed anywhere?

Reply:
I was getting an error without it but the `forc` version update seems to have fixed it. Removed.